src/transformers/generation_logits_process.py

```diff
@@ -702,3 +702,26 @@ class LogitNormalization(LogitsProcessor, LogitsWarper):
     def __call__(self, input_ids: torch.Tensor, scores: torch.Tensor) -> torch.Tensor:
         scores = scores.log_softmax(dim=-1)
         return scores
+
+
+class SuppressTokensAtBeginLogitsProcessor(LogitsProcessor):
+    def __init__(self, begin_supress_tokens, begin_index):
+        self.begin_supress_tokens = list(begin_supress_tokens)
+        self.begin_index = begin_index
+
+    def __call__(self, input_ids, scores):
+        if input_ids.shape[1] == self.begin_index:
+            scores[:, self.begin_supress_tokens] = -np.inf
+
+        return scores
+
+
+class SuppressTokensLogitsProcessor(LogitsProcessor):
+    r""" """
```

Comment:
```suggestion
    r"""This processor can be used to suppress a list of tokens. The processor will set their log probs to `-inf` so that they are not sampled."""
```